File: Listas antigas em C/Lista 1/Questão 15 Lista 1.c
/*
Questão 15 Lista 1
Faça um algoritmo para calcular e imprimir o An de uma P.G. (Progressão Geométrica), segundo a fórmula: an = a1.qn-1;
*/
#include <stdio.h>  
#include <math.h>

int main(void){
	int a1, q, n, An;

	printf("digite a1, q e n\n");
	scanf("%d",&a1);
	scanf("%d",&q);
	scanf("%d",&n);
	An = a1*pow(q, (n-1));
	printf("An = %d", An);
	return 0;
}

File: Lista 1/Questão 15 Lista 1.py
"""
Questão 15 Lista 1
Faça um algoritmo para calcular e imprimir o An de uma P.G. (Progressão Geométrica), segundo a fórmula: an = a1.qn-1
"""
a1 = int(input("Digite o valor de a1: "))
q = int(input("Digite o valor de q: "))
n = int(input("Digite o valor de n: "))
An = a1 * q ** (n - 1)
print(f"An = {An}")
